src/database/ghost.js

```diff
@@ -0,0 +1,26 @@
+/*
+  Tables storing ghost content and its revisions
+*/
+
+import helpers from './helpers'
+
+module.exports = knex => {
+  const h = helpers(knex)
+  return h
+    .createTableIfNotExists('ghost_content', table => {
+      table.increments('id')
+      table.string('folder')
+      table.string('file')
+      table.text('content')
+      table.timestamp('modified_on').defaultTo(knex.fn.now())
+    })
+    .then(() =>
+      h.createTableIfNotExists('ghost_revisions', table => {
+        table.increments('id')
+        table.integer('content_id').references('ghost_content.id')
+        table.string('revision')
+        table.timestamp('created_on').defaultTo(knex.fn.now())
+        table.string('created_by')
+      })
+    )
```

Comment:
@emirotin , you'd rather use `Promise.all` here: creating second table doesn't seem to depend on creating first one.
P.S. I know this is already merged, so not sure whether my review makes sense. )

Reply:
The 2nd one has an FK on the 1st one, I think on PG creating the constraint may fail eventually if the 1st table is not created yet.

It's merged into the temp branch so any significant comments can be addressed there

Reply:
@emirotin @epaminond They indeed to be sequential for Postgres